ocaml/reason/default.nix

```diff
@@ -20,8 +20,8 @@ buildDunePackage {
     then
       builtins.fetchurl
         {
-          url = https://github.com/reasonml/reason/archive/9b459245de02b89a27100d34209a819dd9921de2.tar.gz;
```

Comment:
why? the version was compatible with multicore

Reply:
I think they changed from 5.00 to 5.0 which broke reason